src/app/modules/backbone/components/samples-confirmation-modal/samples-confirmation-modal.component.html

```diff
@@ -7,7 +7,12 @@
         </div>
         <div class="modal-body-content">
             <h3 class="modal-body-content-header" id="modal-title">Order samples</h3>
-            <p class="modal-body-content-message">Are you sure you want to order samples?</p>
+            <p class="modal-body-content-message">Select blend:</p>
+            <div class="modal-body-content-selection-item" *ngFor="let option of selectionOptions">
```

Comment:
I should have been more precise, but we should use the forms module here to track inputs rather than relying on the html